values.yaml

```diff
@@ -50,6 +50,12 @@ st2:
   # If set, st2.password always overrides any existing password.
   # If not set, the password is auto-generated on install and preserved across upgrades.
   #password: Ch@ngeMe
+  # Disable St2 auth secret generation
+  # If set, it requires a the customer to generate a secret with the name of "{{.Release.Name}}-st2-auth"
```

Comment:
Hmm. I was imagining the other direction.

Let the user specify the name of a secret they want to use instead of the `{{.Release.Name}}-st2-auth` secret. If they provide the name of an externally managed secret, then use that AND also skip generation of the `{{.Release.Name}}-st2-auth` secret.

Reply:
@cognifloyd 
I have pushed an update can you please review again?